source/standards/dns-hosting.html.md.erb

```diff
@@ -1,24 +1,23 @@
 ---
-title: DNS hosting
+title: How to manage DNS
 expires: 2018-01-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-To make your services available over the internet they will need DNS records to let people know where they are.
-`service.gov.uk` domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by service teams.
+You need accurate DNS records to ensure your service is available to your users. For example, services on GOV.UK domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by GDS service teams.
 
+As part of your DNS strategy and planning we recommend your:
 
-## Principles
+* DNS entry configuration comes from a version-controlled repository such as GitHub, unless autogenerated
+* overall service strategy includes nameserver availability
 
-- DNS entry configuration should come from a version-controlled repository unless autogenerated
-- Consider nameserver availability as part of your overall service availability
+When you implement your DNS strategy we recommend using:
```

Comment:
This can be stronger I think: "you should use"

Reply:
Agreed, have now changed this.